### DISCUSSION TITLE
Remove weekend from the overview chart

### DISCUSSION THREAD

**Discussion by chripede** [ghostfolio/ghostfolio#1772] [Ideas]
Including the weekend (closed days) in the overview chart results in some seemingly flat days. I think the overview chart should skip Saturday and Sunday.

**Comment by dtslvr** [comment]
Hi @chripede

Ghostfolio is extensively used for cryptocurrencies that are traded 24/7. For this reason, I think that leaving out the weekend is not a favorable option.

**Reply by chripede** [reply]
Good point @dtslvr 
Maybe a setting?

**Reply by dtslvr** [reply]
> Maybe a setting?

Hm, I am not a fan of having a lot of extra settings. Ghostfolio is supposed to be very easy to use.
